CMakePresets.json

```diff
@@ -253,4 +256,4 @@
       "configurePreset": "linux-ninja-gnu"
     }
   ]
-}
+}
```

Comment:
new line missing

Reply:
It's due to JSON formatting rules, which optimize the file to weight as less as possible.

Reply:
I've added the new line.